conf/application.conf

```diff
@@ -33,7 +33,7 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule"
 play.modules.enabled += "uk.gov.hmrc.mongo.play.PlayMongoModule"
 
-play.additional.module = "config.V1Module"
+play.additional.module = "config.V2Module"
```

Comment:
Shall we not leave this at V1 for now for belt and braces? I know we have config to cover this but would feel better if it defaulted to V1 while V1 is live and kicking?